requirements/dev-requirements.txt

```diff
@@ -5,3 +5,4 @@ pep8
 astroid==1.4.8
 pylint==1.6.4
 tox==2.3.1
+mock
```

Comment:
Ah - pytest-mock is arguably buggy here: https://github.com/pytest-dev/pytest-mock/blob/master/setup.py - it only depends on 'mock' on Python 2.6/2.7 - the reasoning for this is likely that mock is indeed in the stdlib in 3.5/3.6 etc - but it requires every import that your using code does to try first for unittest.mock, falling back to mock, because in the stdlib it is at a different path. 

Alternatively - we can do what I've done here, which is to just bring mock in directly - its a rollingback port synchronised with the stdlib, so at least IMO its fine to do that :)